templates/sections/section-person-quote.html.twig

```diff
@@ -5,10 +5,10 @@
 #}
 
 {%- set attributes = attributes.addClass(['centered-container', 'layout--section-person-header']) -%}
-{%- set figure_attributes = create_attribute() -%}
+{%- set quote_attributes = create_attribute() -%}
```

Comment:
where this come from?

Reply:
Post-release merge. Came in after last tag.

Reply:
Probably should have branched off of master if we wanted to do a right hotfix. I could cherry pick it.

Reply:
meh